beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -232,7 +234,7 @@ func (s *Store) verifyAttestation(ctx context.Context, baseState *pb.BeaconState
 	if err != nil {
 		return nil, err
 	}
-	indexedAtt, err := blocks.ConvertToIndexed(ctx, a, committee)
+	indexedAtt, err := attestationutil.ConvertToIndexed(ctx, a, committee)
```

Comment:
```suggestion
	indexedAtt, err := attestationUtil.ConvertToIndexed(ctx, a, committee)
```

Reply:
i think the standard in shared package is lowercase package name like bytesutil

Reply:
yeah i wonder if we should name it `attestation-util` wdyt?

Reply:
<img width="168" alt="Screen Shot 2020-01-29 at 7 00 55" src="https://user-images.githubusercontent.com/5848741/73320408-336af880-4265-11ea-8a0e-d4ef6f99e273.png">
i think it will look strange in this context

Reply:
yeah i think I agree with you here